### DISCUSSION TITLE
企业开源的软件协议模型实践 | 夜天之书

### DISCUSSION THREAD

**Discussion by giscus** [tisonkun/blog#66] [Comments]
# 企业开源的软件协议模型实践 | 夜天之书

我在《企业开源该选什么软件许可证？》一文当中，已经从软件许可证的角度讨论过不同协议对企业开源的意义。 不过，那篇文章主要是从许可证内容出发的，即先给定了协议，分析完协议细节，再判断是否符合企业和计划公开的软件的情形。本文从企业开源的不同诉求和风险出发，结合现存的企业组织实践，重新讨论源码公开第一步就要面临的选择软件协议问题应该如何决策。

https://www.tisonkun.org/2023/02/15/business-source-license/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)
很多时候大家选择视而不见, 并不是不知道这种行为的危害, 而是对自己必须通过海量资料比对研究才能获得相对真实感觉到投入产出比失调….从而放弃治疗, 反正一起烂….
所以, 这时, 聪明人才都在干笨活, 自己亲自吞入海量原始资料, 整理, 给出自己的观点...
从而慢慢形成自己的理念宇宙影响真实世界...
